src/main/java/uk/gov/companieshouse/pscfiling/api/service/FilingDataServiceImpl.java

```diff
@@ -73,4 +75,31 @@ private FilingApi populateFilingData(final FilingApi filing, final String filing
         return filing;
     }
 
+    private void setFilingDescription(FilingApi filing, final PscTypeConstants pscType) {
+        var name = "";
+        switch (pscType) {
+            case INDIVIDUAL:
+                var names = new String[]{(String) filing.getData().get("title"),
+                        (String) filing.getData().get("first_name"),
+                        (String) filing.getData().get("other_forenames"),
+                        (String) filing.getData().get("last_name")};
+                var sb = new StringBuilder();
+                for (String str : names)
+                    if (str != null) {
+                        sb.append(str).append(" ");
+                    }
+                var result = sb.toString();
+                name = result.trim();
+                break;
+            case CORPORATE_ENTITY:
+                // fall through
+            case LEGAL_PERSON:
+                name = (String) filing.getData().get("name");
```

Comment:
Looks good. Suggestion: add a (defensive) `break` here so that a new case added later  doesn't fall through accidentally.

Reply:
ok, added